project/frontend/src/pages/Header.js

```diff
@@ -1,24 +1,105 @@
 import './Header.css';
 
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+
+import Modal from '../components/Modal.js';
+import Loading from '../components/Loading.js';
 
 function Header(props) {
-  return (
-    <div id='header'>
-      {props.isUserLoggedIn ? (
-        <div>Pic</div>
-      ) : (
-        [
-          <button key='1' className='header-button'>
-            Create Account
-          </button>,
-          <button key='2' className='header-button'>
-            Log in
-          </button>,
-        ]
-      )}
-    </div>
-  );
+  let [GoogleAuth, setGoogleAuth] = useState(undefined);
```

Comment:
Do we have to import GoogleAuth?

Reply:
No, I just decided to call that state variable `GoogleAuth`. Perhaps I should not use a capital letter as first letter to make it more clear that it's a variable? `GoogleAuth` is what they were calling that object in the API documentation so I just followed that pattern.

Reply:
The API library is imported in `index.html` inside the `public/` directory. That is were the `window.gapi` object comes from.